external/camel/README.md

```diff
@@ -0,0 +1,20 @@
+# External (Third Party Container) Tests
+Third Party container tests help verify that the AdoptOpenJDK binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries. For each application, we choose to run a selection of their functional tests.
+
+## Running External tests locally
+To run any AQA tests locally, you follow the same pattern:
+Ensure your test machine is set up with test prereqs. For external tests, you do need Docker installed.
```

Comment:
Please restore all of the hyperlinks and ` ` special characters in this document so that `</pathToWhereYouInstalledSDK>` does not dissappear due to being rendered as html.  For the test prereqs hyperlink, if you can link to https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/Prerequisites.md (instead of the one at the openj9 project), , it will be great!

Reply:
Done